lms/static/scripts/frontend_apps/components/test/ContentSelector-test.js

```diff
@@ -0,0 +1,333 @@
+/* eslint-disable new-cap */
+
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+import { act } from 'preact/test-utils';
+
+import mockImportedComponents from '../../../test-util/mock-imported-components';
+import { Config } from '../../config';
+import { PickerCanceledError } from '../../utils/google-picker-client';
+import ContentSelector, { $imports } from '../ContentSelector';
+
+function interact(wrapper, callback) {
+  act(callback);
+  wrapper.update();
+}
+
+describe('ContentSelector', () => {
+  let fakeConfig;
+  let FakeGooglePickerClient;
+
+  beforeEach(() => {
+    FakeGooglePickerClient = sinon.stub().returns({
+      showPicker: sinon.stub(),
+      enablePublicViewing: sinon.stub(),
+    });
+
+    fakeConfig = {
```

Comment:
I'd be careful here, since different components use different subsets of the configuration. Within a given context (eg. the file picker app) the same fake configuration could be used. For different apps (file picker, LTI launch etc.) then different fakes will be needed though.